packages/contracts/package.json

```diff
@@ -22,6 +22,6 @@
     "web3": "^1.2.6"
   },
   "dependencies": {
-    "libsemaphore": "^1.0.5"
+    "libsemaphore": "0.0.17"
```

Comment:
1.x is the audited version but incompatible with the Semaphore repo master branch contract.